pysat/instruments/pysat_testing.py

```diff
@@ -24,15 +23,17 @@
 tags = {'': 'Regular testing data set',
         'no_download': 'simulate an instrument without download support',
         'non_strict': 'simulate an instrument without strict_time_flag',
-        'user_password': 'simulates an instrument that requires a password'}
+        'user_password': 'simulates an instrument that requires a password',
+        'default_meta': 'simulates an instrument using the defualt meta'}
 
 # dictionary of satellite IDs, list of corresponding tags
 # a numeric string can be used in inst_id to change the number of points per day
-inst_ids = {'': ['', 'no_download', 'non_strict', 'user_password']}
+inst_ids = {'': [tag for tag in tags.keys()]}
```

Comment:
```suggestion
inst_ids = {'': [tag for tag in tags]}
```
I don't think you need the .keys part anymore

Reply:
But it is clearer, why would we change it?